docker/docker-compose.yaml

```diff
@@ -28,7 +28,7 @@ services:
 
   test:
     <<: *common
-    command: /bin/bash -xcl "swift test -Xswiftc -warnings-as-errors $${FORCE_TEST_DISCOVERY-} $${SANITIZER_ARG-} $${STRICT_CONCURRENCY_ARG-}"
```

Comment:
6

Reply:
Disabling only 6 isn't great either because we will carry that warning forward. We might have to bite the bullet here and just say this is not an API break since we are using the `@_` API here which was a problem from the beginning.